sfdoc/publish/html.py

```diff
@@ -83,6 +85,8 @@ def create_article_data(self):
             'IsVisibleInCsp': self.is_visible_in_csp,
             'IsVisibleInPkb': self.is_visible_in_pkb,
             'IsVisibleInPrm': self.is_visible_in_prm,
+            "Article_Type__c": self.Article_Type__c,
+            "Topics__c": self.Topics__c,
```

Comment:
Do these fields already exist in production WLMA, or do we need to coordinate the timing of merging this?

Reply:
Yeah, I'm talking to Dave Lea about that in Quip#sfdoc. He says:

> Article_Type__c is available, Topics__c won't be available in production until tomorrow around 8pm EST ish

So yeah, I guess we'll deploy SFdoc on Wednesday morning.